tests/unit/specs/Dictaphone.spec.js

```diff
@@ -33,7 +33,7 @@ describe.only('Dictaphone', () => {
       createObjectURL: jest.fn(),
     };
     window.URL.createObjectURL.mockReturnValue('data...');
-    const wrapper = mount(Component);
+    const wrapper = mount(Dictaphone);
     wrapper.setData({ audioBlob: new Blob(new Uint8Array(null), { type: 'audio/webm' }) });
```

Comment:
When I run the tests this seems to need an additional `await wrapper.vm.$nextTick();` afterwards to work.

Reply:
:heavy_check_mark: